### ISSUE TITLE
Feature request: SOCKSv5 proxy

### ISSUE THREAD

**Issue by omnomnymous** [clbr/fifth#17]
subj

**Comment by clbr** [comment]
Please read doc/proxy - curl supports socks5 among others. This issue
can remain open for having a GUI toggle for them.

**Comment by omnomnymous** [comment]
Yes I'd love to have a GUI toggle. For example this is how my Opera 12 looks like: 
http://i.imgur.com/8dqdfMW.png

**Comment by clbr** [comment]
I'm curious, are you in China or equivalent? I think per-tab proxy use would be so rare, having a button for everyone would waste space for most people.

Per-site proxy is technically possible, though it would be hard to cover all cases if you want to hide your IP. Many sites use CDNs and several subdomains, you'd have to cover them all to avoid leaks.

**Comment by omnomnymous** [comment]
No I'm in Russia where internet censorship gets closer to China every month :)
You're right about subdomains, no need to make per site proxy, one global checkbox is enough.
